### PR TITLE
Use the header x button to close the charmbrowser

### LINKED ISSUE #1103: Revert whitebox close functionality
Hitting the x in the header should close the whitebox instead of just the search field. 
Move the X to the far right.

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes #1103

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the midpoint or store or search or charm details
- the x button should be in the top right of the header
- clicking the x button should close the charmbrowser

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3971/
Test PASSed.

File: jujugui/static/gui/src/app/components/header-search/header-search.js [hunk 1]
```diff
@@ -141,12 +141,9 @@ YUI.add('header-search', function() {
       @method _closeSearch
     */
     _closeSearch: function(active) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/header-search/header-search.js]: active is no longer used.....wonder why lint didn't catch this...

**Comment by hatched** [comment]
:+1: :ok: OMG this is so much better :-)

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3978/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**